gadgets/top_file/gadget.yaml

```diff
@@ -3,56 +3,76 @@ description: Periodically report read/write activity by file
 homepageURL: https://inspektor-gadget.io/
 documentationURL: https://inspektor-gadget.io/docs
 sourceURL: https://github.com/inspektor-gadget/inspektor-gadget/
-toppers:
+datasources:
   file:
-    mapName: stats
-    structName: file_stat
-structs:
-  file_stat:
+    annotations:
+      cli.clear-screen-before: "true"
     fields:
-    - name: pid
-      description: Process ID
-      attributes:
-        template: pid
-    - name: tid
-      description: Thread ID
-      attributes:
-        hidden: true
-        template: pid
-    - name: comm
-      description: Command name
-      attributes:
-        template: comm
-    - name: reads
-      description: Reads by file
-      attributes:
-        width: 8
-    - name: writes
-      description: Writes by file
-      attributes:
-        width: 8
-    - name: rbytes
-      description: Bytes read by file
-      attributes:
-        width: 8
-    - name: wbytes
-      description: Bytes written by file
-      attributes:
-        width: 8
-    - name: t
-      description: Type of file, 'R' for regular files, 'S' for sockets and 'O' for
-        others (including pipes). By default only regular files are shown; use the
-        --all-files option to show all file types.
-      attributes:
-        width: 1
-    - name: file
-      description: File name
-      attributes:
-        width: 30
-    - name: mntns_id
-      description: Mount namespace ID
-      attributes:
-        template: ns
+      pid:
+        annotations:
+          description: Process ID
+          template: pid
+      tid:
+        annotations:
+          description: Thread ID
+          template: pid
+          columns.hidden: true
+      comm:
+        annotations:
+          description: Command name
+          template: comm
+      reads:
+        annotations:
+          description: Reads by file
+          columns.width: 8
+          columns.alignment: right
+      writes:
+        annotations:
+          description: Writes by file
+          columns.width: 8
+          columns.alignment: right
+      rbytes:
+        annotations:
+          description: Bytes read by file
+          columns.width: 8
+          columns.alignment: right
+      wbytes:
+        annotations:
+          description: Bytes written by file
+          columns.width: 8
+          columns.alignment: right
+      t:
+        annotations:
+          description: Type of file, 'R' for regular files, 'S' for sockets and 'O'
```

Comment:
Could he replace the enum values in the eBPF code? So instead of `R` it says `REGULAR`?

Reply:
I think it's a good point, I'm not sure R, S, O are that easy to understand.

Reply:
I'll open an issue for this.